src/elements/button/button.js

```diff
@@ -2,7 +2,7 @@
 'use strict';
 
 angular
-  .module('semantic.ui.elements.button', [])
+  .module('semantic.ui.elements.button', ['semantic.ui.elements.icon', 'semantic.ui.elements.divider'])
```

Comment:
Or I'm not quite not sure by what "should not be here" means, you want me to create a new PR ?